UnityProject/Assets/Scripts/Lighting/LightSource.cs

```diff
@@ -118,7 +120,7 @@ public void ServerChangeLightState(LightState newState)
 
 	private void SyncLightState(LightState oldState, LightState newState)
 	{
-		mState = newState;
+		//mState = newState;
```

Comment:
as per other comments this is actually needed

Reply:
My previous comment on what you said earlier still stands. I'll add them back in just to be sure, but if you're able to explain why I'd much appreciate it.